cmd/git-sync/main.go

```diff
@@ -144,6 +147,16 @@ func main() {
 		flag.Usage()
 		os.Exit(1)
 	}
+	if *flOneTime == false && *flWait != 0 {
```

Comment:
Can you add a separate check - if --wait and --one-time are specified, it should be an error.  We can't make it an error (breaking change) but we can make it log.

Reply:
Thank you for comment. I added it.